assets/pulpcore-api.prep

```diff
@@ -8,5 +8,13 @@ foreground {
 }
 foreground {
   openssl ecparam -genkey -name prime256v1 -noout -out /etc/pulp/certs/token_private_key.pem
+  openssl genrsa -out /etc/pulp/certs/ca.key 2048
+  openssl req -x509 -new -nodes \
+    -key /etc/pulp/certs/ca.key -subj "/CN=pulp/C=US/L=California" \
+    -out /etc/pulp/certs/ca.crt
+  openssl genrsa -out /etc/pulp/certs/pulp_webserver.key 2048
+  openssl req -new -key /etc/pulp/certs/pulp_webserver.key -subj "/CN=pulp/C=US/ST=California" -out /etc/pulp/certs/pulp_webserver.csr
 }
 openssl ec -in /etc/pulp/certs/token_private_key.pem -pubout -out /etc/pulp/certs/token_public_key.pem
+openssl x509 -req -in /etc/pulp/certs/pulp_webserver.csr -CA /etc/pulp/certs/ca.crt -CAkey /etc/pulp/certs/ca.key \
+    -CAcreateserial -out /etc/pulp/certs/pulp_webserver.crt
```

Comment:
That is not how execlineb works...
You need to `foreground` every single command that does not support chain calling, despite the last one.

Reply:
I got confused when reading about `foreground`, thank you for the help!